models/src_recurly.yml

```diff
@@ -149,8 +149,8 @@ sources:
       
       - name: credit_payment_history
         description: A history of outstanding credit balance to pay an invoice, but does include the record of write-offs, credit balance removals, and credit payments later refunded as cash transaction.
-        identifier: "{{ var('credit_payment_history_identifier', 'credit_payment_history') }}"
-        columns:
+        identifier: "{{ var('credit_payment_history_identifier', 'credit_payment_history') }}" 
+        columns: 
```

Comment:
If this model is able to be disabled, we should add the config block to disable it. This goes for all other models that can be disabled.

See [this example](https://github.com/fivetran/dbt_fivetran_log/blob/9f89183075bf03938f81044b673cfb231e353443/models/staging/src_fivetran_log.yml#L138-L139)

Reply:
@fivetran-avinash it doesn't look like the config blocks were added for these models that can be disabled.